### PR TITLE
Update team members

### PR DESCRIPTION
**Pull request by tjouan**

### CODE REVIEW

File: source/stylesheets/corporate/sections/_team.sass [hunk 1]
```diff
@@ -4,8 +4,12 @@
...
 #team
   padding-bottom: $sectionsMargin
-  .col:hover .avatar
-    @include filter(none)
+  .col
+    &:hover .avatar
+      @include filter(none)
+    &:nth-child(4)
+      margin-left: 0
+      margin-right: 0
```
Comment (tjouan) [inline source/stylesheets/corporate/sections/_team.sass]: I'm not really OK with styles in the `&:nth-child(4)` selector.

Without these style changes:
alpinelab_website

With these style changes (9e709327):
alpinelab_website-styles_change

**Comment by tjouan** [comment]
replaced by #7

File: source/stylesheets/corporate/sections/_team.sass (hunk 1 above)
Comment (Spharian) [inline source/stylesheets/corporate/sections/_team.sass]: Care, this is not bootstrap, it's flexbox, that's why your margin: 0 (left, right) is working that way. And the last team member is strangely placed there, no?

Comment (tjouan) [inline source/stylesheets/corporate/sections/_team.sass]: Having worked on bootstrap project just before I mixed up things.
My coworker made same comment, and I'm happy to see it's flexbox.

We applied those changes instead:

dd0599ab7ffd1ba230a24131927302b0fbfeb249
9bb0767c2faa000a58abf1cb11cf9d675e4bd8f6

Last one to improve the justification of wrapped items when less than 3.

Thanks for the comment!

**Pull request closed without merging**